tests/init/neg/Desugar.check

```diff
@@ -0,0 +1,5 @@
+-- Error: tests/init/neg/Desugar.scala:12:61 ---------------------------------------------------------------------------
+12 |    val f: PartialFunction[A, Int] = {case C(_, rhs) => rhs.x} // error
+   |                                                             ^
+   |                   Calling the external method constructor $anon may cause initialization errors. Calling trace:
+   |                    -> val f: PartialFunction[A, Int] = {case C(_, rhs) => rhs.x} // error	[ Desugar.scala:12 ]
```

Comment:
This commit seems to be not associated with your Github account?

Reply:
This one was pushed from git in my local repo

Reply:
Could you please fix that? Make sure every commit is linked to your Github account.

Reply:
It seems that my Github email is private, so I need to reset my git config user.email, but how could I fix this commit to link my account?

Reply:
When you squash the commits, it will use your current settings for the squashed commit, so just make sure they're set correctly before you squash.

Reply:
I have reset my git, but how can I fix these previous commits?

Reply:
You can refer to the following guide:

https://stackoverflow.com/questions/5189560/squash-my-last-x-commits-together-using-git

After that, then force push to your branch.

Reply:
Issue solved. Thanks!